libkbfs/config_local.go

```diff
@@ -897,9 +898,20 @@ func (c *ConfigLocal) EnableJournaling(
 	log := c.MakeLogger("")
 	branchListener := c.KBFSOps().(branchChangeListener)
 	flushListener := c.KBFSOps().(mdFlushListener)
+	// Set the journal disk limit to 10 GiB for now.
+	//
+	// TODO: Base this on the size of the disk, e.g. a quarter of
+	// the total size of the disk up to a maximum of 100 GB.
+	//
+	// TODO: Also keep track of and limit the inode count.
+	var journalDiskLimit int64 = 10 * 1024 * 1024 * 1024
```

Comment:
The reason I'm nervous about 10GB is I've been testing big files, and I can very easily get 3.5 GB of stored bytes in the journal just by copying in a 4 GB file.  And that 3.5 GB stays in there for a while, perhaps minutes, while the first revision flushes.  Probably coalescing could do a better job of breaking up large file writes into smaller revisions, so stored bytes can be removed more quickly. But at the moment, I'm worried it would be pretty easy to fill up 10 GB.

I guess as long as the next PR comes relatively quickly, before we let outsiders start using it, I'm ok with starting here.  But if we ask people to dogfood this version, we should be up front about the limitations.